gems/pending/VixDiskLib/VixDiskLib.rb

```diff
@@ -96,7 +96,7 @@ def self.setup_env
     vars_to_clear.each do |key|
       my_env.delete(key)
     end
-    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"] + ":" + VIXDISKLIB_PATH
+    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"].to_s + ":" + VIXDISKLIB_PATH
```

Comment:
Joe it still works.  Is one method preferred over the other?

On Tue, Aug 18, 2015 at 3:41 PM, Joe Rafaniello <notifications@github.com>
wrote:

> In gems/pending/VixDiskLib/VixDiskLib.rb
> <https://github.com/ManageIQ/manageiq/pull/3932#discussion_r37343625>:
>
> > @@ -96,7 +96,7 @@ def self.setup_env
> >      vars_to_clear.each do |key|
> >        my_env.delete(key)
> >      end
> > -    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"] + ":" + VIXDISKLIB_PATH
> > +    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"].to_s + ":" + VIXDISKLIB_PATH
>
> @movitto <https://github.com/movitto> can we use string interpolation
> here?
>
> my_env["LD_LIBRARY_PATH"] = "#{my_env['LD_LIBRARY_PATH']}:#{VIXDISKLIB_PATH}"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/3932/files#r37343625>.
>



-- 

-- 
Jerry Keselman
Red Hat Virtualization R&Djerryk@redhat.com | p. 212 530 7873